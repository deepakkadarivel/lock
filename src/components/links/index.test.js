import React from 'react';
import { create } from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Links from './index';

describe('Links component', () => {
    test('Renders Links component', () => {
        const component = create(
            <Router>
                <Links />
            </Router>
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
