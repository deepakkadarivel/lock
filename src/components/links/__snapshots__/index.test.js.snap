// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links component Renders Links component 1`] = `
<div
  className="Links"
>
  <ul>
    <li>
      <a
        aria-current="page"
        className="Links-item ripple Links-item--active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        <i
          className="material-icons"
        >
          home
        </i>
        <span
          className="Links-item--text"
        >
          Home
        </span>
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className="Links-item ripple"
        href="/people"
        onClick={[Function]}
      >
        <i
          className="material-icons"
        >
          person_outline
        </i>
        <span
          className="Links-item--text"
        >
          People
        </span>
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className="Links-item ripple"
        href="/doors"
        onClick={[Function]}
      >
        <i
          className="material-icons"
        >
          meeting_room
        </i>
        <span
          className="Links-item--text"
        >
          Doors
        </span>
      </a>
    </li>
    <li>
      <a
        aria-current={null}
        className="Links-item ripple"
        href="/events"
        onClick={[Function]}
      >
        <i
          className="material-icons"
        >
          update
        </i>
        <span
          className="Links-item--text"
        >
          Events
        </span>
      </a>
    </li>
  </ul>
</div>
`;
